lib/src/bazelify/bazelify_config.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:yaml/yaml.dart';
+
+import 'build.dart';
+import 'pubspec.dart';
+
+/// The parsed values from a `bazelify.yaml` file.
```

Comment:
[nit] Should we explain here what types of stuff will be in bazelify.yaml? Will we have that anywhere else?